demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -298,7 +298,7 @@ def is_image_path_valid(self) -> bool:
             bool: True If all links are valid else False.
         """
         invalid_paths = re.findall(
-            r"(\!\[.*?\]|src\=)(\(|\")(https://github.com/demisto/content/(?!raw).*?)(\)|\")",
+            r"(\!\[.*?\]|src\=)(\(|\")(https://github.com/demisto/content/blob/.*?)(\)|\")",
```

Comment:
please add to the test of this validation a case that was failing before and now is ok